### DISCUSSION TITLE
Daily standup (Thursday, March 30)

### DISCUSSION THREAD

**Discussion by DavBot02** [antonieto/ibm-dashboard#80] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Read into NextJS documentation

**What are your priorities for the next 24 hours?**:
 - Start working on figma

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - Research NextJS

**What are your priorities for the next 24 hours?**:
 - Progress in mockups

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto
